lib/segment/src/vector_storage/dense/appendable_dense_vector_storage.rs

```diff
@@ -289,7 +291,8 @@ pub fn open_appendable_in_ram_vector_storage_impl<T: PrimitiveVectorElement>(
 
     let vectors = InRamPersistedVectors::<T>::open(&vectors_path, dim)?;
 
-    let deleted: DynamicMmapFlags = DynamicMmapFlags::open(&deleted_path)?;
+    let populate = true;
+    let deleted: DynamicMmapFlags = DynamicMmapFlags::open(&deleted_path, populate)?;
```

Comment:
Populating memory is of course not the same as forcibly keeping in memory. Is just populating indeed good enough?

Reply:
What I know is that forcing in-ram causing many deployments to go OOM

Reply:
Also previous behavior didn't even try to populate  `DynamicMmapFlags`